draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md

```diff
@@ -107,8 +107,21 @@ The subsection titles in this section attempt to follow the terminology from [@R
 
 ## Zone validator changes
 
+This section covers both the 'online' type of zone validator, such as Zonemaster, and the 'offline full zone' type, such as `validns` and `dnssec-verify`.
+
+Checks for child DNSKEY records based on parent DS records algorithms, and checks for zone RRSIG algorithms based on DNSKEY algorithms, MUST not be applied to algorithm TBD.
+[NOTE: rephrase this in terms of the Zone Signing column at  https://www.iana.org/assignments/dns-sec-alg-numbers/dns-sec-alg-numbers.xhtml ?]
+
+DNSKEY validity checks MAY verify correct DER syntax in DNSKEY Public Key content when algorithm is TBD.
```

Comment:
the MAY here should probably align with, if any, the recommendation for other algorithms?

Reply:
hmm, what do you mean? A document like RFC8080 does not say anything about zone validators